src/components/reusable/notification/notificationPanel.scss

```diff
@@ -0,0 +1,59 @@
+@use '../../../common/scss/global.scss';
+@use '../../../common/scss/mixins.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: block;
+}
+
+.panel {
+  &-wrapper {
+    position: static;
+    display: flex;
+    flex-direction: column;
+    width: 480px;
+    // max-height: 720px;
+    max-height: calc(100vh - var(--kd-header-height) - 16px - 4px);
+    @media (max-width: 495px) {
+      max-width: 320px;
+    }
```

Comment:
Can maybe remove this and just add `max-width: 100%;`

Reply:
> Can maybe remove this and just add `max-width: 100%;`

ok can do that but Marco said max-width for mobile is 320px; Anyway I will add 100%;

Reply:
Mobile needs to fill 100% width. Marco is just using 320 as the baseline in figma, but mobile is a range.

Reply:
ok

Reply:
Done.